.yarnrc.yml

```diff
@@ -6,6 +6,8 @@ conditions:
   USE_ESM:
     default: false
 
+enableConstraintsChecks: true
```

Comment:
This is not needed anymore right? Given that we validate constraints in CI anyway, we can avoid the slowdown on every install.

Reply:
For me, it can be reserved or removed. This check in Yarn 4 is very fast. If it keeps it, we can find that the constraints fail faster.